### ISSUE TITLE
CMake find_package(CrossGuid REQUIRED) error

### ISSUE THREAD

**Issue by bharatph** [graeme-hill/crossguid#34]
The following throws error in macOSX 10.13.3
`
find_package(CrossGuid REQUIRED)
`
I can hint find_package() by providing a path, but would'nt it fail on Windows?

**Comment by bharatph** [comment]
I solved the error by reading the README.md
```sh
mkdir build
cd build
cmake .. -DCMAKE_INSTALL_PREFIX="/usr/local"
make install
```
Request:
The /usr/local can be default prefix without having to give a flag and also an alternative default prefix path can be added to windows for efficiency.

**Comment by graeme-hill** [comment]
I'm guessing that's because @gabyx added this and we would just have to remove it:

```
# Define Install Directories
# Install stuff (default location is not some where on the system! for safty reasons)
if(CMAKE_INSTALL_PREFIX_INITIALIZED_TO_DEFAULT)
    set(CMAKE_INSTALL_PREFIX "${CMAKE_BINARY_DIR}/install" CACHE STRING "Install prefix (e.g. /usr/local/)" FORCE)
endif()
```

Is it really all that unsafe to use the cmake default system dir? I usually assume that if I install something it will put it in a system dir.

**Comment by graeme-hill** [comment]
OK it should be fixed in fad9d395b8291c107152888c0357e3a1a0ffb7d7

I agree that we should not override the default. I even have to `sudo make install` on my system so it's pretty clear that I am installing to a system dir.
